### PR TITLE
Use new rabin js package, improve rabin security level

### PR DESCRIPTION
**Pull request by baryon**

### CODE REVIEW

**Comment by baryon** [comment]
Travis fails until rabin new version was merged

**Review by xhliu — APPROVED** [review]
💯 good work.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "axios": "^0.21.1",
     "glob": "^7.1.6",
     "minimist": "^1.2.5",
-    "rabinsig": "^1.0.0",
+    "rabinsig": "^2.0.0",
```
Comment (xhliu) [inline package.json:33]: This version does not exist, yet. Will update later.

**Comment by xhliu** [comment]
@zhfnjust Can u plz help merge this? New `rabinsig` has to be published first. v1.1.0 or v2.00?
https://app.travis-ci.com/github/sCrypt-Inc/boilerplate/jobs/524351792
https://github.com/sCrypt-Inc/rabin/commit/0c9833c5841572d69d19b355f441cbfb7f0ef24d

**Pull request closed without merging**